src/layup_cmdline/predict.py

```diff
@@ -26,11 +137,105 @@ def main():
     return execute(args)
 
 
+def convert_input_to_jd_TDB(input_str: str) -> float:
+    """
+    Convert a string to a jd_TDB date. The string can be in the format of a
+    Julian date TDB for or a date string in the format YYYY-mm-dd.
+
+    Parameters
+    ----------
+    input_str : str
+        The input string to convert.
+
+    Raises
+    ------
+    ValueError
+        If the input string is not in the expected format, YYYY-mm-dd
+
+    Returns
+    -------
+    float
+        The converted jd_TDB date.
+    """
+    import spiceypy as spice
+
+    try:
+        # Assume that input is a jd_TDB float that was converted to string. Attempt
+        # to convert it back to a float.
+        start_time = float(input_str)
+        start_time = spice.j2000() + start_time / (24 * 60 * 60)
+    except ValueError:
+        # If conversion to float fails, assume that the input was a date string
+        # with a format like YYYY-mm-dd. Convert that to a datetime and then to
+        # a float representation of a jd_TDB date.
+        try:
+            start_time = Time(datetime.strptime(input_str, "%Y-%m-%d"), format="datetime", scale="utc").tdb.jd
```

Comment:
change scale to "tdb" and then this will work as intended so that 

 layup predict  main.py -s 2000-01-01
 gives  
 
 Start time: 2451544.5
End time: 2451558.5